torchgeo/models/swin.py

```diff
@@ -39,8 +59,8 @@ class Swin_V2_B_Weights(WeightsEnum):  # type: ignore[misc]
     .. versionadded:: 0.6
     """
 
-    NAIP_RGB_SATLAS = Weights(
-        url="https://huggingface.co/torchgeo/swin_v2_b_naip_rgb_satlas/resolve/main/swin_v2_b_naip_rgb_satlas-685f45bd.pth",  # noqa: E501
+    NAIP_RGB_SI_SATLAS = Weights(
+        url="https://huggingface.co/allenai/satlas-pretrain/resolve/main/aerial_swinb_si.pth",  # noqa: E501
```

Comment:
We usually include the first 8+ characters of the sha256sum in the suffix of the URL so that functions like https://pytorch.org/docs/stable/hub.html#torch.hub.load_state_dict_from_url can automatically verify the download checksum.